package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "prestart": "if [ ! -d node_modules ]; then npm install --quiet; fi",
     "start": "concurrently --raw --kill-others 'gulp' 'gulp worker:dev' 'maildev' './scripts/agendash-dev'",
-    "start:skipBower": "gulp --skipBower",
+    "start:skipBower": "concurrently --raw --kill-others 'gulp --skipBower' 'gulp worker:dev' 'maildev' './scripts/agendash-dev'",
```

Comment:
This is already on master: https://github.com/Trustroots/trustroots/commit/727eb23815de153580c05a44ea5e1c6a9827fcb0

You can do `git fetch origin && git rebase origin/master` to get it.